src/core/thread.cc

```diff
@@ -1,3 +1,8 @@
+// If _FORTIFY_SOURCE is defined than longjmp ends up using longjmp_chk
```

Comment:
might want to s/than/then/ ?

yeah, better off removing the `set_source_files_properties(..)` command. since we undefine `_FORTIFY_SOURCE` in the source file.

Reply:
Yes good call. Done